### DISCUSSION TITLE
Incorrect post request?

### DISCUSSION THREAD

**Discussion by bakerhaider** [swellstores/community#483] [Help] [answered]
Hi i have made a custom field in my order model called 'cusomer_file' which im now trying to make a post request to from my client buti get this error: 
![e4fe7d8b6e3d0a0777a8558cf3f44a9c](https://github.com/swellstores/community/assets/33936834/93eedec7-9053-43f4-8b83-40e08fb328a5)

**Comment by logeshswell** [comment, marked as answer]
I assume that you use swell-js (frontend API). You are expected to open up write permission for the custom field as follows:
https://developers.swell.is/guides/opening-content-permissions#using-input-fields

You can also update the custom field with `content` namespace which will be editable by default.

**Reply by logeshswell** [reply]
Can you confirm that you use swell-js instead of swell-node?

**Reply by logeshswell** [reply]
I also believe that your code may throw an error since you are sending a request to an unexposed endpoint `'/orders/cusomer_file'`. So, it could be corrected as follows:
```js
const res = await swell.post('/orders', {
  cusomer_file: 'eee'
});
```

But, the above code will work only in swell-node (Backend API), as you don't have provisions to make POST /orders request in swell-js.
https://developers.swell.is/backend-api/orders/create-an-order

On a side note, you don't need to open up content permissions for backend APIs.
